examples/website/src/shared/teasers/TeaserLink.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {ReactNode} from 'react'
+import {Link, Route} from '../route/routeContext'
+
+export interface TeaserLinkProps {
+  readonly url: string
+  readonly route?: Route
+  readonly children: ReactNode
+  readonly className?: string
+  readonly isPeerArticle?: boolean
+}
+
+export const TeaserLink = ({
```

Comment:
You don't need this new component. The `<Link>` component already accepts the attributes `href` and with `target` you can control where it should open. If `<Link>` has `route===undefined` it takes the `href` attribute.